### ISSUE TITLE
As a user, I would like to have separate import for construction standards and for use types

### ISSUE THREAD

**Issue by AlexJew** [architecture-building-systems/CityEnergyAnalyst#2779]
**Describe the feature you would like to have**
Hello,

I was wondering if it was possible to have separate imports for construction standards and use types database? Currently CEA enables to separately import archetypes, assemblies and components, but I believe that a more interesting separation would be between construction standards and use types. This separation corresponds to a real separation between the building as an object and the building as a program / a function.

**Describe the end purpose of this feature**
In my work, I would like to simulate different "behaviours" for a same building stock. This is to understand the impact of change in the occupant lifestyle on the energy demand.

**Describe the end-user of this feature**
A practitionner

**Provide Aditional information if possible**
These are the available choices:

![Screen Shot 2020-06-15 at 12 29 57](https://user-images.githubusercontent.com/32110882/84652759-111c0380-af04-11ea-8706-9869e696547a.png)

**Comment by daren-thomas** [comment]
@AlexJew - I'm not quite sure what you're trying to do. If I understand correctly, you'd like to run multiple scenarios and change the use types of the buildings for each such scenario (e.g. between OFFICE and RESIDENTIAL) right? I think that can be done by just changing the use type of the building in the Input Editor. 

What am I missing?

**Comment by AlexJew** [comment]
Hello @daren-thomas, in my case, I was studying two different user behaviours in multi-dwelling units. Indeed, in Portugal, there is the energy poverty issue which forces certain residents to reduce their energy demand for space heating thus affecting their thermal comfort. Therefore, I simulated the energy demand for space heating and cooling with different schedules for setpoint and setback modes, plus different shares of air-conditionned areas. I could have built two separate MULTI_RES (for instance a MULTI_RES_POOR and a MULTI_RES_STANDARD), however I opted instead in having two different databases and a single USE_TYPE for the buildings (one _typology.dbf_ file). 

But it is a minor request, and maybe I should've used two different _typology.dbf_ files and a single database with differentiated use types for those whose operation mode changed depending on the thermal comfort scenario considered.
